content/widgets.md

```diff
@@ -58,6 +58,18 @@
 * Update the preferred content size
 * Wait for `viewWillTransition(to:with:)` to be called and update UI accordingly
 
+## Detecting if Device is Locked
+
+1. The first time your main app starts create a dummy file (let's call it ```ProtectionMonitor.dummy```) in a shared group container which is accessible for both the main app and the widget.
+	* You can create this file with ```NSFileManager.createFile(atPath:, contents:, attributes:)```
+2. Make sure the file is protected by iOS Data Protection.
+	* You can ensure by passing the dictionary ```{NSFileProtectionKey: NSFileProtectionComplete}``` as the attributes parameter in the above method
```

Comment:
You can ensure this...

...paramater when creating the file
(if we were to add a bullet point later, we'll likely miss this reference which then becomes confusing)